### PR TITLE
Update PublishData.json for Preview2

### PR DESCRIPTION
**Pull request by agocke**

### CODE REVIEW

**Comment by agocke** [comment]
@jasonmalinowski Forgot to update this for preview2. Can you take a look?

File: build/config/PublishData.json [hunk 1]
```diff
@@ -35,6 +35,13 @@
             "vsix": [ "https://dotnet.myget.org/F/roslyn/vsix/upload" ],
             "channels": [ "dev16.0p1" ]
         },
+        "dev16.0-preview2-vs-deps": {
+            "nugetKind": ["Shipping", "NonShipping"],
+            "version": "2.11.*",
+            "nuget": [ "https://dotnet.myget.org/F/roslyn/api/v2/package" ],
+            "vsix": [ "https://dotnet.myget.org/F/roslyn/vsix/upload" ],
+            "channels": [ "dev16.0p2" ]
```
Comment (jasonmalinowski) [inline build/config/PublishData.json]: You need to update the master-vs-deps one now to say dev16.0p3.

Comment (agocke) [reply build/config/PublishData.json]: Yes, thanks

Comment (agocke) [reply build/config/PublishData.json]: @jasonmalinowski Look good now?

**Pull request merged**